src/main/java/io/antmedia/muxer/MuxAdaptor.java

```diff
@@ -118,6 +118,13 @@
 	protected boolean dashMuxingEnabled;
 	protected boolean objectDetectionEnabled;
 
+	protected ConcurrentHashMap<String, Boolean> isHealthCheckGoing = new ConcurrentHashMap<>();
```

Comment:
Please make names more descriptive in order to let anyone understand the code easily.